### PR TITLE
added data to explore.ts

### PR DESCRIPTION
**Pull request by zatzby**
I have updated the explore.ts file mainly, also added a NostriChat.svelte component (currently have it inactive)..

I am not sure if I have the NostriChat component setup correctly (pretty sure I don't actually). 

Looks like it needs a group id hex, although maybe that is accomplished through the NIP-07 sign in? Or do I provide my own - that would be specific to whynostr?

 It looks like NIP28 is the correct NIP for group chat. Here's link

[https://github.com/nostr-protocol/nips/blob/master/28.md](url) 
I am going to check it out in more detail asap.

Do you have any thoughts/ideas on that in the meantime?

Also, any feedback on the new explore page. I have been updating the sites and categories. Do they seem cohesive and correct? 

I am trying to add more from the Nostrverse (plus orange pill friendly) but hoping to not get too off track, so I hope some of the new sites I have added don't detract (such as the Other Stuff category or some links within Commerce)

### CODE REVIEW

File: src/lib/components/Header.svelte [hunk 1]
```diff
@@ -12,6 +12,6 @@
             <a class="navLink no-underline md:text-xl font-semibold {$page.url.pathname.includes('explore') ? 'active' : ''}" href="/explore">Explore</a>
             
         </div>
-        <ProfileMenu on:logout on:login />
+         <ProfileMenu on:logout on:login />
```
Comment (erskingardner) [inline src/lib/components/Header.svelte:15]: Remove the space here.

File: src/lib/components/NostriChat.svelte [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<script>
+    const groupID = "<GROUP_ID_IN_HEX_FORMAT>";
```
Comment (erskingardner) [inline src/lib/components/NostriChat.svelte:2]: Yup – in this case you need to replace the group id in hex with the hex of the actually NIP-28 chat room. You can create those in this app: https://www.nostrchat.io/

**Comment by erskingardner** [comment]
The site looks great – I love how much stuff there is on the explore page now. 🎉

File: src/lib/components/Header.svelte (hunk 1 above)
Comment (zatzby) [reply src/lib/components/Header.svelte:15]: on it.

File: src/lib/components/NostriChat.svelte (hunk 2 above)
Comment (zatzby) [reply src/lib/components/NostriChat.svelte:2]: Very cool! Thank you for sharing how that is done.

**Comment by zatzby** [comment]
Thank you for feedback Jeff! Going to make adjustments now then merge.

**Review by zatzby — COMMENTED** [review]
Thank you for feedback, adjustments have been made.

**Pull request merged**